src/docs/operator-guides/concepts/index.md

```diff
@@ -1,337 +1,271 @@
 
-### Availability plan & availability exception
 
-An availability plan is a recurring plan for the dates and times when a
-listing is available. For example, a provider can mark their listing to
-be available by default on weekdays (Mon–Fri) from 8am to 8pm, and
-unavailable during night time and on weekends.
+### Transaction process 
 
-An availability exception is a deviation from the availability plan. For
-example, the provider can make an exception and also mark selected
-Fridays unavailable, or make certain weekends available for overnight
-bookings.
+Transaction process defines the rules of what and how transactions happen on the marketplace. Anytime two users interact on a Flex marketplace, what steps are available to them, when they are available, and what happens as a result of these steps is enclosed in the transaction process. 
 
-### Banned user
+You can have different transaction processes for different ways of transacting, like renting and buying products, in the same marketplace. You can see the transaction processes of your marketplace in Console’s “Build” section.
 
-Banning a user means removing the user and all of the user’s listings
-from a marketplace due to inappropriate behaviour. The email with which
-a banned user registered to the marketplace can't be used to create new
-accounts. The user data is only visible when it’s linked to (only ID +
-banned status are shown).
+[Click here](https://www.sharetribe.com/docs/concepts/transaction-process/) to learn more about transaction processes and for tips on how desing your own.
 
-### Booking
+### Microcopy editor 
 
-A reservation of a listing for a specific time.
+The Microcopy editor lets you edit your marketplace’s microcopy from Console's Build section. Use the Microcopy editor to edit the short written texts scattered around your marketplace, like button labels, error messages, or help texts. 
 
-### Closed listing
+[Click here](https://www.sharetribe.com/docs/operator-guides/how-to-use-microcopy-editor/) to learn more about the Microcopy editor and how to use it. 
 
-A published listing can be closed by its author or the marketplace
-admin. Closed listings are not returned in search results or public
-listing queries. This doesn't mean the listings are deleted: they can
-still be accessed as own listings or with a direct URL.
+### Pages editor 
 
-The motivation for closing a listing is not to delete it, but to
-(temporarily) stop advertising it. A closed listing can be opened, which
-makes it published again.
+The Pages editor lets you create, modify, and delete static content pages on your marketplace. It comes pre-loaded with four static content pages: the About page, the Landing page, the Privacy policy, and the Terms of use page. Each of these can be editor with the Pages editor, or additional content pages can be created. You can find the editor in Console's Build section.
```

Comment:
"With Pages you can create, modify, and delete static content..."

"Each of these can be editor with Pages, or..."